### PR TITLE
move-ownership-to-npm-debitoor

### PR DESCRIPTION
**Pull request by hilleer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "express-timeout-handler",
-  "version": "2.2.0",
```
Comment (ebdrup) [inline package.json]: :-(

Comment (jonatanpedersen) [reply package.json]: No no. We just need Oliver to add us as collaborators in NPM

Comment (jonatanpedersen) [reply package.json]: We are not moving it to @debitoor

Comment (jonatanpedersen) [reply package.json]: @Oligrand Help :)

Comment (Oligrand) [reply package.json]: You are added now @jonatanpedersen 🚀

Comment (hilleer) [reply package.json]: Yay, thanks @Oligrand. Closing this PR then :)

Comment (Oligrand) [reply package.json]: Cool, and nice to see this module is getting some love! @hilleer

Comment (jonatanpedersen) [reply package.json]: @Oligrand I still don't have access

Comment (jonatanpedersen) [reply package.json]: Yay, it works now :)

Comment (Oligrand) [reply package.json]: that's weird, I can see you:

image

Comment (Oligrand) [reply package.json]: Ah ok it works now lol

**Pull request closed without merging**